7-toolchain/Dockerfile.fedora

```diff
@@ -1,24 +1,23 @@
-FROM fedora:27
-LABEL MAINTAINER Honza Horak <hhorak@redhat.com>
+FROM registry.fedoraproject.org/f27/s2i-core:latest
```

Comment:
We should probably move this to a later Fedora base if it is already being touched.

Reply:
Fedora 27 is latest release having gcc 7, so we need to keep f27.